index.js

```diff
@@ -235,6 +249,22 @@ function Physics (mcData, world) {
               vel.z *= physics.soulsandSpeed
             } else if (block.type === webId) {
               entity.isInWeb = true
+            } else if (block.type === bubblecolumnId) {
+              const drag = block.getProperties().drag
```

Comment:
As drag is the only property, you can get it more efficiently using block.metadata (0 or 1), it avoids some string manipulations